pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -1017,6 +1040,11 @@ def multi_parity_multi_round_seq(ancilla_qubit_names,
                         if first_readout[round] is True:
                             all_pulses[-1]['name'] = f'first_ro_{round}_loop' + \
                                 f'_{m}_tomo_{t}'
+                            # all_pulses[-1]['ref_pulse'] = 'segment_start'
+                            # all_pulses[-1]['pulse_delay'] = round*parity_seperation \
+                            #                                 + (1+rounds)*\
+                            #                                 m*parity_seperation +\
+                            #                                 (16/2.4)*150e-9
```

Comment:
needed ?

Reply:
I don't know...that function is super messy..

Reply:
Not sure about the commented code here. Christian wrote it. I would either leave the code as it is or delete both the measure and sequence functions from master.

Reply:
Yep, I deleted both. We will deal with it when Christian gets back.